docs/rackhd/authentication.rst

```diff
@@ -0,0 +1,174 @@
+Accessing RackHD APIs with Authentication
+-----------------------------------------
+
+With authentication is enabled in config.json configure file for an endpoint,
+see `Setting up HTTP/HTTPS endpoint`_ and `Authentication`_, authentication is needed for
+accessing the API group associated with the specified endpoint.
+
+This section describes how to access APIs that need authentication.
+
+Setup endpoints to enable authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please refer to `Setting up HTTP/HTTPS endpoint`_ on how to setup endpoints. Simply put,
+the following endpoint configuration will be a good start.::
+
+    "httpEndpoints": [
+        {
+            "address": "0.0.0.0",
+            "port": 8443,
+            "httpsEnabled": true,
+            "proxiesEnabled": false,
+            "authEnabled": true,
+            "routers": "northbound-api-router"
+        },
+        {
+            "address": "172.31.128.1",
+            "port": 8080,
+            "httpsEnabled": false,
+            "proxiesEnabled": false,
+            "authEnabled": false,
+            "routers": "southbound-api-router"
+        }
+    ],
+
+The first endpoint represents a HTTPs service listening at port 8443 that serves northbound APIs, which are
+APIs that being called by users. Noticed that authEnabled is set to true means the authentication is needed
+for accessing northbound APIs.
+
+The second endpoint represent a HTTP service listening at port 8080 that serves southbound APIs, which are
+called by nodes. Authentication should NOT be enabled for southbound APIs in order for PXE to work fine.
+
+Setup credentials and token for authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please refer to `Authentication`_ on how to setup endpoints.
+
+Copy following items to config.json if they are not there yet.::
+
+    "authUsername": "admin",
+    "authPasswordHash": "KcBN9YobNV0wdux8h0fKNqi4uoKCgGl/j8c6YGlG7iA0PB3P9ojbmANGhDlcSBE0iOTIsYsGbtSsbqP4wvsVcw==",
+    "authPasswordSalt": "zlxkgxjvcFwm0M8sWaGojh25qNYO8tuNWUMN4xKPH93PidwkCAvaX2JItLA3p7BSCWIzkw4GwWuezoMvKf3UXg==",
+    "authTokenSecret": "RackHDRocks!",
+    "authTokenExpireIn": 86400,
+
+Following above configuration, the default username/password for login are set to be admin/admin123,
+and the token will expire 24 hours after it's generated.
+
+Please follow the instruction on `Change the default password`_ if there is a need to change the
```

Comment:
@yyscamper , yes, a section called "Change the default password" is in the configuration.rst.